content/en/news/releases/1.22.x/announcing-1.22/change-notes/index.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Istio 1.22.0 Change Notes
+linktitle: 1.22.0
+subtitle: Minor Release
+description: Istio 1.22.0 release notes.
+publishdate: 2024-05-13
+release: 1.22.0
+weight: 10
+aliases:
+    - /news/announcing-1.22.0
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definition). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** usage of `values.istio_cni` in favor of `values.pilot.cni`
+  ([Issue #49290](https://github.com/istio/istio/issues/49290))
+
+## Traffic Management
+
+- **Improved** `ServiceEntry` with `resolution: NONE` to respect `targetPort`, if specified.
+This is particularly useful when doing TLS origination, allowing to set `port:80, targetPort: 443`.
+If undesired set `--compatibilityVersion=1.21` to revert to the old behavior, or remove the `targetPort` specification.
+
+- **Improved** XDS generation to send less resource when possible, sometimes omitting a response entirely.
+This can be disabled by the `PILOT_PARTIAL_FULL_PUSHES=false` environment variable, if necessary.
```

Comment:
```suggestion
- **Improved** XDS generation to do utilize fewer resources when possible, sometimes omitting a response entirely.
  This can be disabled by the `PILOT_PARTIAL_FULL_PUSHES=false` environment variable, if necessary.
```